notification/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+services:
+  notification-service:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8083:8083"
+    depends_on:
+      - kafka
+      - mysql
+    environment:
+      SPRING_DATASOURCE_URL: jdbc:mysql://mysql:3306/notification_db
+      SPRING_DATASOURCE_USERNAME: root
+      SPRING_DATASOURCE_PASSWORD: password
+      KAFKA_BOOTSTRAP_SERVERS: kafka:9092
```

Comment:
this might conflict with the kafka broker on chat service. Not worth changing yet, we might need to remove it from docker compose but keep it in application.properties